GravityPDF/gravity-pdf
# Move our standard Uninstaller into Gravity Forms uninstall page

## Description

Gravity Forms has migrated all their add-on uninstaller to their Settings -> Uninstall page. This PR moves Gravity PDF's uninstaller to the same location.

Resolves #1191 

## Testing instructions
1. Go to Settings->Uninstall
2. Select Gravity PDF's Uninstall button
3. Confirm you want to proceed
4. Plugin is uninstalled and deactivated

## Screenshots 
![Screen Shot 2021-03-12 at 11 12 44 am](https://user-images.githubusercontent.com/2918419/110872422-e42a3f00-8323-11eb-8869-6e0dab2cb876.png)

## Checklist:

## Additional Comments
